staticfiles/admin/js/nav_sidebar.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+{
+    const toggleNavSidebar = document.getElementById('toggle-nav-sidebar');
+    if (toggleNavSidebar !== null) {
+        const navLinks = document.querySelectorAll('#nav-sidebar a');
+        function disableNavLinkTabbing() {
+            for (const navLink of navLinks) {
+                navLink.tabIndex = -1;
+            }
+        }
+        function enableNavLinkTabbing() {
+            for (const navLink of navLinks) {
+                navLink.tabIndex = 0;
+            }
+        }
+
+        const main = document.getElementById('main');
+        let navSidebarIsOpen = localStorage.getItem('django.admin.navSidebarIsOpen');
+        if (navSidebarIsOpen === null) {
+            navSidebarIsOpen = 'true';
+        }
```

Comment:
Instead of checking for null and then setting the default value, you can use the logical OR operator to simplify the code. If the value is null, the OR operator will return the second operand. For example, you can write `let navSidebarIsOpen = localStorage.getItem('django.admin.navSidebarIsOpen') || 'true';`